cpu/esp32/Makefile.dep

```diff
@@ -57,6 +57,7 @@ ifneq (,$(filter esp_spiffs,$(USEMODULE)))
     USEMODULE += vfs
     export SPIFFS_STD_OPTION = -std=c99
     INCLUDES += -I$(RIOTBASE)/sys/include
+    # Why is this include here?
```

Comment:
@gschorcht Why is this `INCLUDES +=` needed? The "use vfs" should have the headers to be included anyways. Is it some build system weirdness?

Reply:
@jcarrano Honestly? I can not remember anymore. Probably, these includes were needed during the port sometime in between for the compilation before I could solve all dependencies.

Reply:
@jcarrano BTW, I have tested it without this include, it seems to be compilable.

Reply:
I could solve this problem. I will provide a pull request which removes these INCLUDES.